src/libdrakvuf/drakvuf.c

```diff
@@ -602,6 +620,138 @@ unicode_string_t* drakvuf_read_unicode_va(vmi_instance_t vmi, addr_t vaddr, vmi_
     return drakvuf_read_unicode_common(vmi, &ctx);
 }
 
+
+/*
+ * The following function has been taken directly
+ * from LibVMI. IMHO we need to include in LibVMI
+ * a function for reading an unicode string
```

Comment:
Sorry but taking code directly from LibVMI is not good because the licenses are incompatable. If a new function in LibVMI is all thats needes please do that instead.

Reply:
Ok, so we end up with 2 choices:

1. Make the appropriate changes to LibVMI and send the PR
2. Make our own Drakvuf function (quicker than the previous option I guess)

Which one do you prefer? I'm opened to opinions...

Reply:
I would say option 1. I maintain LibVMI as well so it's going to take the same amount of time ;)

Reply:
I know, I know... :)
I sent, a few minutes ago, the PR libvmi/libvmi#720 to the LibVMI repository with the new function suitable for using in our code.

Reply:
The aforementioned LibVMI PR has been merged into master so as soon as the Drakvuf LibVMI branch gets updated I'll finish this PR